### PR TITLE
Sprint 32 TLT-2009 Check for available iSites to import when rendering the course dashboard

### PR DESCRIPTION
**Pull request by douglashall**
- Disable the Course iSites Content Migration link on the Course Dashboard when there are not any available iSites to migrate.
- Also display an informational message describing this state.

### CODE REVIEW

**Comment by elliottyates** [comment]
@elliott

File: isites_migration/views.py [hunk 1]
```diff
@@ -29,7 +29,11 @@ def index(request):
         )
         return redirect('isites_migration:index')
 
-    processes = Process.objects.filter(name='isites_migration.jobs.migrate_files').order_by('-date_created')
+    processes = Process.objects.filter(
+        name='isites_migration.jobs.migrate_files',
+        details__at_canvas_course_id=canvas_course_id
```
Comment (elliottyates) [inline isites_migration/views.py:34]: I can't find where the `at_canvas_course_id` field is set in the `details` blob in our isites_migration app, can you point it out to me?

**Comment by elliottyates** [comment]
This also fixes an interaction with the table when there are no active jobs, right?

File: isites_migration/views.py (hunk 1 above)
Comment (douglashall) [inline isites_migration/views.py:34]: This is django-pgjson query syntax.

http://djangonauts.github.io/django-pgjson/#_querying

**Comment by douglashall** [comment]
Yes, I also fixed a bug where the "Begin iSties Import" button would not become enabled after selecting a keyword from the dropdown if there weren't any active migrations.

File: isites_migration/views.py (hunk 1 above)
Comment (douglashall) [inline isites_migration/views.py:34]: details['canvas_course_id'] is set in Process.enqueue()

https://github.com/Harvard-University-iCommons/django-icommons-common/blob/master/async/models.py#L74

Comment (elliottyates) [inline isites_migration/views.py:34]: My bad. I was looking at [django-jsonbfield](https://github.com/HearstCorp/django-jsonbfield]).

**Comment by elliottyates** [comment]
:+1:

**Pull request merged**